src/scenes/Lobby/Settings/index.tsx

```diff
@@ -0,0 +1,83 @@
+import Header from '../../../components/Header';
+import Background from '../../../components/Background';
+import GameCard from '../../../components/GameCard';
+import { AlertTriangle } from 'react-feather';
+import './Settings.css';
+
+type Game = {
+  id: number;
+  text: string;
+  src: string;
+  backgroundColor: string;
+};
+
+interface SettingsProps {
+  gameList: Game[];
+  mainPage: () => void;
+  updateGameList: React.Dispatch<React.SetStateAction<Game[]>>;
+}
+
+export default function Settings({
+  gameList,
+  mainPage,
+  updateGameList,
+}: SettingsProps) {
+  const updateSelection = (id) => {
+    console.log(id);
+    const newID = id < 1000 ? id + 1000 : id - 1000;
```

Comment:
Não entendi isso aqui, podes explicar?

Reply:
Estás usando o valor do id pra identificar se um jogo foi selecionado ou não?

Reply:
precisamente. se o ID for >= 1000 o jogo está deselecionado. Se for menor que isso (0, 1, 2...) está selecionado.